src/main/modules/webpack/govukFrontend.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
```

Comment:
I used import/export in my code. There are very very few places where i used require because i had to. There are some pros and cons to using either import or require but let's be consistant in our project. 

Below copied from stackOverflow:
ES6 Imports:

You can use named imports to selectively load only the pieces you need. That can save memory.
Import can be asynchronous (and in current ES6 Module Loader, it in fact is) and can perform a little better.

Reply:
just realised this is js file.

Reply:
Agreed. Might also be nice to create a barrel for this.

Reply:
On another note, those they need to be js? it doesn't seem massiviely consistent, but if the webpack needs to be JS, then it's different story.